implementations/micrometer-registry-graphite/src/test/java/io/micrometer/graphite/GraphiteTagNameMapperTest.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright 2020 Pivotal Software, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.graphite;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.config.NamingConvention;
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests for {@link GraphiteTagNameMapper}.
+ *
+ * @author Jon Schneider
+ * @author Johnny Lim
+ * @author Andrew Fitzgerald
+ */
+class GraphiteTagNameMapperTest {
+    private final GraphiteTagNameMapper nameMapper = new GraphiteTagNameMapper();
+    private final SimpleMeterRegistry registry = new SimpleMeterRegistry();
+    private final NamingConvention namingConvention = new GraphiteNamingConvention();
+
+    @Test
+    void simpleName() {
+        Meter meter = registry.counter("a.simple.counter");
+        assertThat(getName(meter)).isEqualTo("a.simple.counter");
```

Comment:
Now that tags aren't part of the metric name, I think it makes sense to use them as is instead of camel casing